dv-export/python/query-changes/README.md

```diff
@@ -10,25 +10,160 @@ The application performs the following operations:
+This placeholder is replaced with actual values from the DV export files. For examples, see the `queries` path. For more information about list queries, see [Data Query API Reference](https://docs.visier.com/developer/apis/data-model-query/swagger/current/index.html#/Query/Query_List:~:text=Query%20a%20list%20of%20details).
 
 Command-line argument examples:
-- `-b 7000005 -d 7000006 -q queries/employee.json`: Export Employee changes for the Applicant object between data versions `7000005` and `7000006`.
-- `-e 92edafd0-22aa-11ef-9027-6fa2c7d9d5d2 -q queries/productivity.json`: Export Productivity changes for the Productivity object using an existing export with the UUID `92edafd0-22aa-11ef-9027-6fa2c7d9d5d2`. You can find `export_uuid` in logs after running a DV export job.
+- `-b 7000005 -d 7000006 -m restate -q queries/employee.json`: Fetches the full history (restate) for the Employee if the EmployeeID is among the changed values between data versions `7000005` and `7000006`.
+- `-e 92edafd0-22aa-11ef-9027-6fa2c7d9d5d2 -m last -q queries/productivity.json`: Fetch the last changes for the `Productivity` analytic object if the `EmployeeID` is among the changed values between data versions, using the existing export with the UUID `92edafd0-22aa-11ef-9027-6fa2c7d9d5d2`.
+- `-e 92edafd0-22aa-11ef-9027-6fa2c7d9d5d2 -m restate -q queries`: Fetch the full history (restate) for all queries in the `queries` directory using the existing export with the UUID `92edafd0-22aa-11ef-9027-6fa2c7d9d5d2`.
+
+## Data Samples:
+
+Let's consider the timeline for analytic object `Employee` and the `DataVersion` as follows.
+
+`V#`: These labels represent different versions of the Employee analytic object,
+indicating changes that have occurred over time.
+
+`DV#`: These labels represent different data versions in the Visier project.
+
+```plaintext
+Employee1:   ------V1--V2-------------------------V3------V4------->
+Employee2:   ---------------------V1---V2-------------V3----------->
+DataVersion: -DV1-----------DV2------------DV3-----------------DV4->
+```
+
+The same data is presented in a table format with dates:
+
+| Date       | Employee1 | Employee2 | DataVersion |
+|------------|-----------|-----------|-------------|
+| 2024-01-01 |           |           |             |
+| 2024-01-05 |           |           | DV1         |
+| 2024-01-10 | V1        |           |             |
+| 2024-01-15 | V2        |           |             |
+| 2024-01-25 |           |           | DV2         |
+| 2024-02-01 |           | V1        |             |
+| 2024-02-05 |           | V2        |             |
+| 2024-02-10 |           |           | DV3         |
+| 2024-02-20 | V3        |           |             |
+| 2024-03-01 |           | V3        |             |
+| 2024-03-10 | V4        |           |             |
+| 2024-03-25 |           |           | DV4         |
+
```

Comment:
@Shmalikov Is this showing changes to specific EmployeeIDs over time (eg, an employee record changes)?